sgkit_bgen/tests/test_bgen_reader.py

```diff
@@ -0,0 +1,30 @@
+import numpy.testing as npt
+from sgkit_bgen import read_bgen
+
+
+def test_read_bgen(shared_datadir):
+    path = shared_datadir / "example.bgen"
+    ds = read_bgen(path, chunks=(100, 500))
```

Comment:
Is `example.bgen` more than 500 samples?  I think a test with smaller chunk size than the number of samples would be a good idea if not.

Reply:
Parameterized for a variety of chunk sizes. a7eb8437647d7eb2ec3e0b37e74f511739aba190